src/app/component/eco-news/create-news/create-news.component.html

```diff
@@ -68,7 +68,7 @@ <h3 class="textarea-title">Content</h3>
             <button
                 type="button"
                 (click)="navigateByUrl('/news')">Cancel</button>
-            <button type="button" (click)="navigateByUrl('create-news/preview')">Preview</button>
+            <button type="button" (click)="goToPreview()">Preview</button>
             <button type="submit" [disabled]="!createNewsForm.valid">Publish</button>
```

Comment:
same as line 71

Reply:
Not my scope

Reply:
but you added (click)="goToPreview", so fix at least, what you added

Reply:
fixed